src/tutorial/runtime/runtime-configuration.rst

```diff
@@ -253,8 +253,8 @@ Start, Stop, Restart
 
```

Comment:
There is a warning at the top of the file saying 

> This section of the tutorial still uses a command from the Rose project to copy workflow source files.

I think it can be removed?

Reply:
Yes, as can the TODO - platformize everything